database/migrations/2016_08_13_102611_add_profile_picture_to_users.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddProfilePictureToUsers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('profile_picture')->default('missing.png');;
```

Comment:
string name, as in _profile_pictue_ or that it defaults to missing?
In the version I'm working on, it defaults to null but I've left it as _profile_picture_